lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -199,7 +199,7 @@ class IndexQueryTestCase(ModuleStoreTestCase):
     NUM_PROBLEMS = 20
 
     @ddt.data(
-        (ModuleStoreEnum.Type.mongo, 8),
+        (ModuleStoreEnum.Type.mongo, 9),
```

Comment:
Late-breaking update: query count went up by one due to fetching the course in seq_module